site/en/docs/android/custom-tabs/translate-language/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: "layouts/doc-post.njk"
+title: Translate your content
+seoTitle: "Guide: Localize Content in a Custom Tab"
+date: 2023-04-21
+description: How to translate content in a Custom Tab.
+authors:
+  - alexmkatz
+---
+
+When a user opens a Custom Tab, the content in that tab can be translated for their locale. This guide
+explains how to implement automatic translation of Custom Tab content.
+
+## Implementation
+
+Use the following code to add translation to your Custom Tabs intent builder:
+
+```
+CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
+    .setTranslateLanguage(true)
+    .setTranslateLocale(Locale locale)
+    .build();
+```
+
+The intent builder takes the user's locale as input. For best user experience, we recommend using the full
+language code (such as `pt-BR` or `sr-latn-RS`).
+
+<!--Note from Kevin: Check whether this auto-translates, or asks the user whether they want it translated. Test
```

Comment:
I couldn't get this to work, so it'd be good to check with eng.

Reply:
Do you happen to know Jinsuk's Github username?

Reply:
We have this documented somewhere and I verified it with eng recently. We just need to use a supported language code. I know that sr-latn-SR isn't on the list. Let me see if I can find it.

Reply:
Here's the list. Eng helped me verify this with Web Store source code in August. https://developer.chrome.com/docs/webstore/i18n/#choosing-locales-to-support

Reply:
how's this?

Reply:
You example language code need to be one of the ones in this list.

Reply:
Latest commit links to the list instead. Is that insufficient?

Reply:
Updated.